### PR TITLE
Add docs methods in SDK to extract context and platform properties

### PR DESCRIPTION
**Pull request by nodo**
[#154396204]

Signed-off-by: Sarah Connor <sconnor@pivotal.io>

### CODE REVIEW

File: creating.html.md.erb [hunk 1]
```diff
@@ -802,8 +802,25 @@ type DashboardUrlGenerator interface {
...
+groups to jobs for the deployment (`deploymentInstanceGroupsToJobs`). This
+mapping must be provided by the service author. This function will not address
+job level properties for the generated instance groups; these properties must
+also be provided by the service author. For an example implementation see the
+[job mapping in the Kafka example
+adapter](https://github.com/pivotal-cf-experimental/kafka-example-service-adapter/blob/ffe85fecc2d9cf1b663fab418875e67ddcf3327f/adapter/generate_manifest.go#L14-L20).
+
+The SDK provides also the methods `ArbitraryContext` and `Platform`. These are
+used to extract the `context` property and the `platform` property, within
+`context` from the request parameters. The context in the response is a feature of [OSBAPI spec
```
Comment (deniseyu) [inline creating.html.md.erb]: This suggests that the structure of the request params is `context.platform` and `context.context`. Is that accurate?

**Comment by deniseyu** [comment]
Approved

**Comment by deniseyu** [comment]
@snneji @ghanna2017 @jncd please merge into master only

**Comment by snneji** [comment]
Thanks @nodo

**Pull request merged**